rest_service.go

```diff
@@ -78,7 +78,7 @@ func (s *Service) App() *gimlet.APIApp {
 	app.AddRoute("/process/{id}/metrics").Version(1).Get().Handler(s.processMetrics)
 	app.AddRoute("/process/{id}/signal/{signal}").Version(1).Patch().Handler(s.signalProcess)
 	app.AddRoute("/process/{id}/logs").Version(1).Get().Handler(s.getLogs)
-	app.AddRoute("/process/{id}/signal-trigger/{trigger-id}").Version(1).Patch().Handler(s.registerSignalTriggerID)
+	app.AddRoute("/process/{id}/trigger/{trigger-id}").Version(1).Patch().Handler(s.registerSignalTriggerID)
```

Comment:
I worry that this is ambiguous with post-run triggers... should we put /trigger/signal/{id} to clarify?

Reply:
I changed this to `/trigger/signal`.